examples/jetstream/workqueue-stream/java/Main.java

```diff
@@ -0,0 +1,171 @@
+package example;
+
+import io.nats.client.*;
+import io.nats.client.api.StreamConfiguration;
+import io.nats.client.api.ConsumerConfiguration;
+import io.nats.client.api.StreamInfo;
+import io.nats.client.api.StreamState;
+import io.nats.client.api.RetentionPolicy;
+import io.nats.client.api.AckPolicy;
+
+import java.io.IOException;
+import java.time.Duration;
+import java.nio.charset.StandardCharsets;
+
+public class Main {
+  public static void main(String[] args) {
+    String natsURL = System.getenv("NATS_URL");
+    if (natsURL == null) {
+        natsURL = "nats://127.0.0.1:4222";
+    }
+
+    // Initialize a connection to the server. The connection is AutoCloseable on exit.
+    try (Connection nc = Nats.connect(natsURL)) {
+
+      // ### Creating the stream
+      // Initialize the `JetStreamManagement` context for managing JetStream
+      // assets.
+      JetStreamManagement jsm = nc.jetStreamManagement();
+
+      // Define the stream configuration, specifying the retention policy
+      // as `WorkQueue` and create it.
+      StreamConfiguration sc = StreamConfiguration.builder()
+        .name("EVENTS")
+        .subjects("events.>")
+        .retentionPolicy(RetentionPolicy.WorkQueue)
+        .build();
+
+      jsm.addStream(sc);
+      System.out.printf("Created stream %s\n", sc.getName());
```

Comment:
Maybe this instead to show the info from the result of the addStream call?
```
StreamInfo si = jsm.addStream(sc);
System.out.printf("Created stream %s\n", si.getConfiguration().getName());
```

Reply:
Good idea, fixed.